design/FY2019/Heat Emissions from Buildings to Ambient.md

```diff
@@ -0,0 +1,216 @@
+
+## Overview
+
+Quantifying heat emissions from buildings and its spatiotemporal distribution have significant implications on urban environment and climate studies. Knowing how much heat is released into atmosphere can help quantify the magnitude of its impact on urban microclimate, e.g., urban heat island effect and climate change. Spatial and temporal distribution of heat emissions can serve to locate hot spots in space and time, therefore useful for prioritizing recourse to mitigate heat emissions from buildings.
+
+We propose a bottom-up approach to calculate heat emissions from buildings with any building energy modeling programs. The scope is restricted to report the buildings’ heat directly discharged to their ambient air. As illustrated in Figure 1, we propose to include heat releases from three levels of building components: (1) building envelope - exterior surface convection and radiative heat transfer to the ambient air, (2) zone - exhaust air or exfiltration to ambient, (3) HVAC system - relief/exhaust air from AHUs, and (4) HVAC system - heat rejected by air-cooled condensers or central plants, including cooling towers of water-cooled chillers, gas-fired boilers, furnaces and water heaters. It should be noted that any under-ground zone or envelope is excluded, since heat from those spaces is rejected to ground instead of to ambient air. 
+
+The heat emission we calculate and report measures the heat transfer from buildings to their ambient environment, and can have both positive and negative values. The positive values indicate the building injects heat to the environment, while the negative values indicate the building extracts heat from the environment.
+
+For heat transfer via air flow, we calculate and report both the sensible and latent heat emissions.
+
+![Diagram](bldgreleasedheat.png)
+
+**Figure 1. Building released heat compositions**
+
+**1.	Heat emission from building envelope**
+
+Exterior surfaces emit heat to ambient air in the form of convection and long-wave radiation, which could be absorbed by particles and gas molecules in air. Figure 2 illustrates the heat balance of an exterior surface. The internal sources modify the outside surface temperature through envelope conduction, along with incident solar radiation, which triggers the heat exchange with the ambient air. _Surface Outside Face Convection Heat Gain Rate_ and _Surface Outside Face Thermal Radiation to Air Heat Transfer Coefficient_ have already been reported. We would calculate the _Surface Outside Face Thermal Radiation to Air Heat Transfer Rate_ as:
+
+Q<sub>lwr, surf</sub> = Hr * A * (T<sub>surf</sub> - T<sub>out</sub>) &nbsp;&nbsp; Eq. (1),
+
+where **Hr** is the surface outside face thermal radiation to air heat transfer coefficient, **A** is the surface area, **T<sub>surf</sub>** and **T<sub>out</sub>** are the surface and outdoor air temperature.
+
+![Diagram](bldgenvelopheatbalance.png)
+
```

Comment:
I am not sure if solar reflection is a part of heat emission from building envelope. However, solar reflection is caused by a building.

Reply:
The intention is to include the thermal impact of buildings to their ambient so the net thermal radiation to the air of surfaces is included.